service/src/main/java/bio/terra/pipelines/dependencies/rawls/RawlsService.java

```diff
@@ -57,7 +57,7 @@ public WorkspaceDetails getWorkspaceDetails(
         () ->
             rawlsClient
                 .getWorkspacesApi(accessToken)
-                .listWorkspaceDetails(workspaceNamespace, workspaceName, fields)
+                .listWorkspaceDetails(workspaceNamespace, workspaceName, fields, null)
```

Comment:
this new null is for an optional `userProject` param, which is not something we care about: `When specified, bill google storage requests to this project`